src/clientdlg.cpp

```diff
@@ -1159,7 +1159,25 @@ void CClientDlg::Connect ( const QString& strSelectedAddress, const QString& str
         catch ( const CGenErr& generr )
```

Comment:
`linux/sound.cpp`, `android/sound.cpp`, `ios/sound.mm`, `src/soundbase.cpp`, `mac/sound.cpp`, `windows/sound.cpp` - these all throw CGenErr from multiple places.

`src/socket.cpp` throws the error, too - and it's caught here.

Reply:
Hmm, you think we should add a subtype of error to distinguish the audio error case?